src/xdocs/cmdline.xml.vm

```diff
@@ -169,27 +169,33 @@ java -D&lt;property&gt;=&lt;value&gt;  \
     <section name="Download and Run">
       <p>
         It is possible to run Checkstyle directly from the JAR file using
-        the <code>-jar</code> option. Download latest
+        the <code>-jar</code> option.
+       </p>
+       <p>
+        Follow the steps mentioned below to run Checkstyle locally:
+       </p>
+        <p>
+        1. Download latest
         <a href="https://github.com/checkstyle/checkstyle/releases/download/checkstyle-${projectVersion}/checkstyle-${projectVersion}-all.jar">
-        checkstyle-${projectVersion}-all.jar</a>.
-        An example of run would be:
-      </p>
+           checkstyle-${projectVersion}-all.jar</a>.
+        </p>
+        <p>
+        2. Create a configuration file required to run Checkstyle.
```

Comment:
This example does not need config file creation as it uses already prepared embedded config.

Attention to `/` before name of file. It signals to load it from classpath/jar too.

Reply:
Sounds good. I may remove the 2. point I've mentioned. Any more suggestions ?

@rnveach Any idea on why all the jobs are failing? The site has been built successfully though.

Reply:
Look at CI. `xml-maven-plugin` is failing from your changes.

Reply:
Please remove

Reply:
@romani The changes have been applied now! Kindly take a look.